README.adoc

```diff
@@ -1,122 +1,337 @@
 
-. Change to the project directory.
+. *Change to the project directory.*
 +
 .Example
+[source,shell]
 ----
 cd docs_dir
 ----
 
-. Re-associate your files with the proper repository.
-
-.. _If your docs project is its own Git repo_, change the project's remote origin Git repository.
+. *Re-associate your files* with the proper repository.
+[upperalpha]
+.. _If your docs project is its own Git repo_, change the boilerplate project's remote origin Git repository to your own.
 +
+.Example
+[source,shell]
 ----
 git remote set-url origin git://github.com:your-project/your-repo-name.git
 ----
 +
 Replace this URI with your own.
 
-.. _If your docs project is part of a parent directory's Git repo_, remove local Git files so your new docs source will be absorbed into the parent repo.
+.. _If your docs project is part of a parent directory's Git repo_, remove _local_ Git files (for the LDCMF repo you just cloned) so your new docs source will be absorbed into the parent repo.
 +
+.Example
+[source,shell]
 ----
 rm -rf .git
 ----
 +
 Now your new directory/files will appear as uncommitted additions to the parent project (`git status`).
 
-. Run Bundler to install dependencies.
+. *Run Bundler* to install dependencies.
 +
+[source,shell]
 ----
 bundle
 ----
 +
 If an error indicates Bundler is not installed, run `gem install bundler`, then repeat this command.
 
-. Rename this README so _init_ does not overwrite it.
+. *Rename this README* so the upcoming _init_ operation does not overwrite it.
 +
 .Example
+[source,shell]
 ----
 mv README.adoc README.meta.adoc
 ----
 +
 This file may confuse future users; we advise removing it before sharing your LDCMF project.
 
-. Edit your project metafile.
+. *Edit the metafile* for your new LDCMF project.
 +
 In your favorite text/code editor, open `data/meta.yml` and fill out the required info, then *save the file*.
 
-. Run LiquiDoc with the LDCMF initialization config.
+. *Initialize your custom environment.*
+[lowerroman]
+.. Generate an _initialization config_.
 +
+[source,shell]
 ----
-bundle exec liquidoc -c _init/init.yml
+bundle exec liquidoc --template _init/templates/init.yaml --data data/meta.yml --output _init/init.yml
 ----
 +
-This procedure customizes a few starter files based on `meta.yml`.
+This command invokes barest LiquiDoc, which performs a one-off template parsing.
 
-. You can now review your `README.adoc` and `content/index.adoc` files, editing and populating them with custom content.
+.. Generate _infrastructure files_ based on your custom preferences.
++
+[source,shell]
+----
+bundle exec liquidoc -c _init/init.yml
+----
+
+=== Build the Docs
+
+. You can now *review the generated `README.adoc`*. +
+The new README file is oriented for _your LDCMF application's_ future users (and you!).
+
+. *Perform the first build of your seedling docset*, as instructed in the new README. +
 
```

Comment:
I'm a bit confused here, and I'm not sure if it's just because I'm testing.

This what I specified in `meta.yml`....which of course created the corresponding instructions in the new README. 
![image](https://user-images.githubusercontent.com/30160806/47524058-6d717180-d84e-11e8-947b-45e253e61275.png)

But it didn't actually create that new repo. I'm still in my bootstrap-test repo on my local machine. So following the README instructions isn't going to take me anywhere. What am I missing?

Reply:
Also, as a user I am wondering how I will regenerate these docs for my documentation project as time goes on.

Reply:
That's it. You're IN your new documentation project. This part is hard to explain but it's easier than simple to do -- it's already done. The whole objective is to convert the cloned repo into your new documentation project. You don't need to be "taken anywhere" :-)